src/networking/RoleService.ts

```diff
@@ -53,7 +53,7 @@ export default class RoleService extends ModelService {
           WHERE_OPERATOR.eq,
           doc(NetworkService.getDb(), COLLECTION_NAME.Person, personId)
         ),
-        where("title", WHERE_OPERATOR.eq, ROLE_TITLE.COUNCILMEMBER),
+        where("title", WHERE_OPERATOR.in, [ROLE_TITLE.COUNCILMEMBER, ROLE_TITLE.COUNCILPRESIDENT]),
```

Comment:
we will have to be careful about our usage of 'in' (and its counterpart, not-in), as we are limited by Firebase to only 1 of them per query. Just a reminder.